### PR TITLE
Add JDK15 JVMTI Version

### PR DESCRIPTION
**Pull request by JasonFengJ9**
* Defined `JVMTI_VERSION_15`;
* Updated `jvmtiGetVersionNumber()` & `jvmtiHookGetEnv()`;
* Minor refactoring.

Reviewer: @gacholio 
FYI: @DanHeidinga 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: runtime/include/jvmti.h.m4 [hunk 1]
```diff
@@ -48,7 +48,9 @@ JNIEXPORT jint JNICALL Agent_OnAttach(JavaVM* vm, char *options, void *reserved)
 #define JVMTI_VERSION_9  0x30090000
 #define JVMTI_VERSION_11 0x300b0000
```
Comment (pshipton) [inline runtime/include/jvmti.h.m4:50]: Pls remove this blank line.

Comment (JasonFengJ9) [reply runtime/include/jvmti.h.m4]: Yeah, removed.

**Comment by gacholio** [comment]
jenkins test sanity win jdk11

**Comment by pshipton** [comment]
jenkins test sanity zlinux jdknext

**Comment by gacholio** [comment]
I've seen enough from the windows builds that it's not worth relaunching it when the build issue gets fixed.

**Comment by keithc-ca** [comment]
I suspect the Windows failure was due to https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/307 which I am reverting.

**Comment by keithc-ca** [comment]
Jenkins test sanity win jdk11

**Comment by pshipton** [comment]
Reverted, see https://github.com/eclipse/openj9/issues/9889

**Pull request merged**